api/src/main/java/org/apache/iceberg/transforms/TransformUtil.java

```diff
@@ -62,6 +62,14 @@ static String humanTimestampWithoutZone(Long timestampMicros) {
     return ChronoUnit.MICROS.addTo(EPOCH, timestampMicros).toLocalDateTime().toString();
   }
 
+  static String humanTimestampNanoWithZone(Long timestampMicros) {
+    return ChronoUnit.NANOS.addTo(EPOCH, timestampMicros).toString();
```

Comment:
Looks like this is incorrect. The value is `timestampMicros`.

Reply:
The value is in nanoseconds; the name was an erroneous copy and paste, now fixed.